assets/quick-add.js

```diff
@@ -86,5 +87,24 @@ if (!customElements.get('quick-add-modal')) {
       galleryList.setAttribute('role', 'presentation');
       galleryList.querySelectorAll('[id^="Slide-"]').forEach(li => li.setAttribute('role', 'presentation'));
     }
+
+    updateImgSizes() {
+      const mediaGallery = this.modalContent.querySelector('[id^="MediaGallery-"]');
+      
+      if (mediaGallery.dataset.desktopLayout === 'columns') {
```

Comment:
This could be an early return instead of wrapping everything 🤔 Though I wonder if it should live more where the function is called. Otherwise we're running this `updateImgSize()` function every time we fetch though it's only needed for a specific use case.
Im gonna take a bit more time to think about it 👍